app/authorisation/login/loginInteractor.js

```diff
@@ -0,0 +1,58 @@
+const ValidationError = require('../../errors/validationError');
+const NotFound = require('../../errors/notFound');
+const ApiError = require('../../errors/apiError');
+
+class LoginInteractor {
+    constructor({
+        presenter,
+        validator,
+        userRepository,
+        bcrypt,
+        responseBuilder,
+        jwt
+    }) {
+        this.presenter = presenter;
+        this.validator = validator;
+        this.userRepository = userRepository;
+        this.bcrypt = bcrypt;
+        this.responseBuilder = responseBuilder;
+        this.jwt = jwt;
```

Comment:
Since we do not really care what lib do we use, we should hide them behind corresponding services: `AuthTokenService`, `PasswordHashService` etc.